openroad.bzl

```diff
@@ -297,6 +297,19 @@ def get_entrypoint_cmd(
 
     return cmd
 
+def filter_out_outputs(outs, stage):
+    """
+    Filters out log and report files from stage's output.
+
+    Args:
+      outs: dictionary of lists with paths to output files for each flow stage
+      stage: name of the stage
+
+    Retruns:
+      list with filtered output files
+    """
+    return [i for i in outs.get(stage, []) if len(i) > 4 and i[-4:] not in (".log", ".rpt")]
```

Comment:
Should we use https://bazel.build/rules/lib/core/string#endswith?

> sub 	[string](https://bazel.build/rules/lib/core/string.html); or [tuple](https://bazel.build/rules/lib/core/tuple.html) of [string](https://bazel.build/rules/lib/core/string.html)s; required
The suffix (or tuple of alternative suffixes) to match.

Reply:
endswith doesn't jell with "in (...)", so it would have to be map... Good 'nuf for now.